src/ui/components/App.vue

```diff
@@ -71,6 +71,7 @@ import Exporters from '@/ui/components/app/Exporters';
 import { trackPage } from '@/analytics';
 import ReportProblemDialog from './app/ReportProblemDialog';
 import MainContent from './app/MainContent';
+import appProperties from '../../../package.json';
```

Comment:
```suggestion
import { repository } from '../../../package.json';
```

Since you need this object only for the `repository` field, you can extract the `repository` field into variable with the same name